dtc-parser.y

```diff
@@ -169,6 +169,8 @@ devicetree:
 			 */
 			if (!($<flags>-1 & DTSF_PLUGIN))
 				ERROR(&@2, "Label or path %s not found", $1);
+			else if (strchr($1 + 1, '/'))
```

Comment:
I don't think this condition is quite right.  This determines that the reference is *not* a pure label, but it won't distinguish between an absolute path reference and a label-relative path reference.

Reply:
Ah, yes. With the lexer changes, it's indeed not correct anymore. I defined a helper now to check it correctly and used it here.